polaris-core/src/main/java/org/apache/polaris/core/PolarisConfigurationStore.java

```diff
@@ -69,6 +69,16 @@ public interface PolarisConfigurationStore {
 
     if (config.defaultValue instanceof Boolean) {
       return config.cast(Boolean.valueOf(String.valueOf(value)));
+    } else if (config.defaultValue instanceof Short) {
+      return config.cast(Short.valueOf(String.valueOf(value)));
+    } else if (config.defaultValue instanceof Integer) {
+      return config.cast(Integer.valueOf(String.valueOf(value)));
+    } else if (config.defaultValue instanceof Long) {
+      return config.cast(Long.valueOf(String.valueOf(value)));
+    } else if (config.defaultValue instanceof Float) {
```

Comment:
Similarly, I think we can leave off short. I don't know why you would want a config to be a float and not a double

Reply:
Updated to remove float